tests/Integration/Behaviour/Features/Scenario/Customer/customer_management.feature

```diff
@@ -38,7 +38,22 @@ Feature: Customer Management
       | lastName  | Napoler                     |
       | email     | naapoler.dev@prestashop.com |
       | password  | PrestaShopForever1_!        |
-    Then I should be returned an error message 'Customer with email "naapoler.dev@prestashop.com" already exists'
+    Then I should be returned an error message 'Registered customer with email "naapoler.dev@prestashop.com" already exists'
+
+  Scenario: Ability to create a guest customer, even if registered customer with that email exists
+    When I create a customer "CUST-3" with following properties:
+      | firstName | Mathieu                     |
+      | lastName  | Napoler                     |
+      | email     | naapoler.dev@prestashop.com |
+      | password  | PrestaShopForever1_!        |
+      | isGuest   | true                        |
+    When I query customer "CUST-3" I should get a Customer with properties:
```

Comment:
Optional and not important, just scenario reads better IMO
```suggestion
    And I query customer "CUST-3" I should get a Customer with properties:
```

Reply:
Done ✅